antora/docs/modules/ROOT/templates/policy_configuration.hbs

```diff
@@ -278,9 +278,23 @@ as a data source, for example:
 
 NOTE: If the data source contains policy rules, those will be ignored.
 
-NOTE: The data source must provide the full set of required data values. These
-are all the values defined in
+NOTE: If you replace the default data source entirely you must provide the full set of required data values.
```

Comment:
Should there be a comma? `entirely, you`

Reply:
Done.